Source/CommandsWindow.cpp

```diff
@@ -81,6 +84,7 @@ CommandsWindow::CommandsWindow(BString& title)
 						.SetInsets(0)
 						.Add(fBrowseButton = new BButton("Browse" B_UTF8_ELLIPSIS, new BMessage(kBrowseCommandAction)))
 						.AddGlue()
+						.Add(fEditButton = new BButton("Edit" B_UTF8_ELLIPSIS, new BMessage(kEditCommandAction)))
 					.End()
```

Comment:
I'm still debating about the text of the button.  It might be better to be more explicit and say "Edit file".  We can probably leave it like this for now until I get around to adding translation support.

Reply:
What do you think about icon buttons. A folder icon for the "Browse" and a text icon + pen for the "Edit"?
Those could be put to the right of the BTextControl which would free up the space below as a positive side-effect.

Reply:
I'm not against having icons but I was more concerned about making it clear that the file would be opened, instead of opening an editor that could be used to edit the current command line arguments.  Maybe I'm overthinking it or not giving the users enough credit.

Reply:
Not sure it's really a problem. I think the user will click on the icon and sees what happens and will know now what it exactly does. It's non-destructive, so nothing can really go wrong. Plus we can add a tooltip.